CPT331.WebAPI/Api/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace CPT331.WebAPI.Api
+{
+    public class TestController : ApiController
+    {
+        // GET api/<controller>
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET api/<controller>/5
+        public string Get(int id)
+        {
+            return "value1";
+        }
+
+        // POST api/<controller>
+        public void Post([FromBody]string value)
```

Comment:
This is just a template ApiController using default conventions. This convention states that the Method (Get/Put/Post/Delete) used is derived from the method name (Whatever the method name starts with).

I think you really only need the attribute [HttpGet] to achieve the same result as your comment.